data-prepper-api/src/test/java/com/amazon/dataprepper/model/log/JacksonLogTest.java

```diff
@@ -49,4 +50,16 @@ public void testBuilder_withNonLogMetadata_throwsIllegalArgumentException() {
         assertThat(logBuilder, is(notNullValue()));
         assertThrows(IllegalArgumentException.class, logBuilder::build);
     }
+
+    @Test
+    public void testBuilder_usesLogEventType_withDefaultTimestamp() {
+        final Log log = JacksonLog.builder()
+                .withEventType("test")
+                .getThis()
+                .build();
+
+        assertThat(log, is(notNullValue()));
+        assertTrue(log.containsKey("event_timestamp"));
```

Comment:
FYI, Hamcrest has a `hasKey()` matcher. This would yield: `assertThat(log, hasKey("event_timestamp"));`. It's not necessary, but I think it is nice.